CONTRIBUTING.md

```diff
@@ -263,6 +263,19 @@ request has been approved and all checks have passed, click the
 button. The resulting commit message will be based on the pull request's title and
 description.
 
+### Reverting a pull request
+
+If a merged pull request needs to be undone, for reasons such as breaking the build, the
+standard process is to revert it through the GitHub interface.
```

Comment:
I would link this to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request

Reply:
Done.